README.md

```diff
@@ -1,162 +1,251 @@
-| evaluatorCacheSize | int | Defines the authorino evaluator cache size in megabytes, defaults to 1 if not specified  |
-| listener | [Listener](#listener)  | Specification of the authorization service (gRPC interface) |
-| oidcServer | [OIDC Server](#oidcserver) | Specification of the OIDC service |
+```sh
+make install
+```
 
-#### Listener
+3. Deploy the Operator
 
-Specifies the details to communicate to an external authenticiation server, check out [Authorino's architecture](https://github.com/Kuadrant/authorino/blob/main/docs/architecture.md) for further details.
+```sh
+make deploy
+```
 
-| Field | json field | Type | Info  |
-|----------|---------|:----:|---------|
-| Port     | port | number | Specify to port to the server, if not speficied it will default to port `50051` |
-| Tls     | tls | object | [TLS](#tls) |
+<details>
+  <summary><i>Tip:</i> Deploy a custom image of the Operator</summary>
+  <br/>
+  To deploy an image of the Operator other than the default <code>quay.io/kuadrant/authorino-operator:latest</code>, specify by setting the <code>OPERATOR_IMAGE</code> parameter. E.g.:
 
-#### OIDCServer
+  ```sh
+  make deploy OPERATOR_IMAGE=authorino-operator:local
+  ```
+</details>
 
-Specifies the details to communicate to an OIDC server, check out [Authorino's architecture](https://github.com/Kuadrant/authorino/blob/main/docs/architecture.md) for further details.
 
-| Field | Type | Description  |
-|----------|:----:|---------|
-| port | number | Specify to port to the server, if not speficied it will default to port `8083` |
-| tls | [TLS](#tls) | Specification of the TLS certificate |
+### Installing via OLM
 
-#### TLS
+To install the Operator using the [Operator Lifecycle Manager](https://olm.operatorframework.io/), you need to make the Operator CSVs available in the cluster by creating a `CatalogSource` resource.
 
-Defines the TLS certificate to a server
+The bundle and catalog images of the Operator are available in Quay.io:
 
-| field | Type | Description  |
-|----------|:----:|---------|
-| enabled | boolean | Defines the TLS certificate is enabled, which will make the cert secret mandatory (Defaults to true) |
-| certSecretRef   | [v1.LocalObjectReference](https://v1-15.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/#localobjectreference-v1-core) | The reference to the secret that contains the TLS certificates `tls.crt` and `tls.key`.|
+<table>
+  <tbody>
+    <tr>
+      <th>Bundle</th>
+      <td><a href="https://quay.io/kuadrant/authorino-operator-bundle">quay.io/kuadrant/authorino-operator-bundle</a></td>
+    </tr>
+    <tr>
+      <th>Catalog</th>
+      <td><a href="https://quay.io/kuadrant/authorino-operator-catalog">quay.io/kuadrant/authorino-operator-catalog</a></td>
+    </tr>
+  </tbody>
+</table>
 
-Example:
-```yaml
+1. Create the namespace for the Operator
```

Comment:
In bare kubernetes, OLM is not installed. Maybe add a section to install the OLM? `operator-sdk` CLI has a command for that:

```
operator-sdk olm install
```

Reply:
There's a [Quick Start](https://olm.operatorframework.io/docs/getting-started/) section in the OLM page we linked above. It covers installing OLM in a cluster, general instructions to install operators using OLM, as well as the prerequisite of having the SDK binary itself somewhere locally (another step that might not be obvious for bare Kubernetes users, IMO).

So, given that there are not one but actually 2 steps to get OLM in the K8s cluster, the fact that in some cases OLM will be already available in the cluster (e.g. OpenShift users, K8s users already familiar with Operator SDK and OLM), plus the existence of a good page dedicated to such part of the instructions, I think I prefer saving this step and keep our sequence simple.

Thanks for the good tip anyway, @eguzki! I'll leave it here together with this reply of mine for the record, in case somebody else is wondering.